sdk/storage/storage-blob/src/ContainerClient.ts

```diff
@@ -1285,4 +1297,24 @@ export class ContainerClient extends StorageClient {
       }
     };
   }
+
+  private getContainerNameFromUrl(): string {
```

Comment:
Please log an issue for possible future improvements using a `blobUriBuilder` that supports extracting blob specific components from a url.

Reply:
Sure